html/ui/js/brs.js

```diff
@@ -550,6 +550,38 @@ var BRS = (function(BRS, $, undefined) {
         }
     };
 
+	BRS.activateAccount = function activate_acc(){
+		$.ajax({
+			type: 'GET',
+			url: 'https://explorer.burstcoin.network/api_activate.php?pkey=' + BRS.publicKey + '&acc=' + BRS.accountRS ,
```

Comment:
IMHO the host should be configurable...

Reply:
Will try and add it to brs-default.properties

Reply:
Implemented in settings and PR updated